webclient/app/src/app/features/apps/entities/DefaultEntities.js

```diff
@@ -14,11 +16,103 @@ const emptyEntity = {
     name: "none",
     fields: [],
     relationships: [],
-    isNewEntity: false,
     position: {
         positionX: 0,
         positionY: 0
     }
 };
 
-export {newEntity, emptyEntity};
+function lowerFirstChar(s) {
+    return (s && s[0].toLowerCase() + s.slice(1)) || "";
```

Comment:
```suggestion
    return (string && string[0].toLowerCase() + string.slice(1)) || "";
```

Finde ich ein wenig übersichtlicher ;)